examples/demo.yaml

```diff
@@ -8,7 +8,7 @@ apiVersion: v1
 kind: PersistentVolume
```

Comment:
Can we add a namespace resource in the demo manifest so when we apply it, the `chaos-demo` namespace is created (and deleted when we delete it)? Otherwise it means we need to manually create it but as a user I'd expect the demo manifest to work without any additional steps than applying it.

Reply:
I didn't even realize you could add namespaces via manifests, I've always done it manually. I'll add that